model/tensorflow_hub/dffml_model_tensorflow_hub/tfhub_utils.py

```diff
@@ -0,0 +1,316 @@
+"""Tokenization classes implementation.
+This code is forked from:
+https://github.com/google-research/bert/blob/master/tokenization.py.
```

Comment:
https://github.com/google-research/bert/blob/master/LICENSE

Reply:
Let's list in the README everything that was copied from other places with links to those places and their licenses.